APIKit/RequestType.swift

```diff
@@ -97,10 +102,14 @@ public extension RequestType {
             if parameters.count > 0 {
                 components.percentEncodedQuery = URLEncodedSerialization.stringFromDictionary(parameters)
             }
-            
+
         default:
             do {
-                URLRequest.HTTPBody = try requestBodyBuilder.buildBodyFromObject(parameters)
+                if parameters.count > 0 {
+                    URLRequest.HTTPBody = try requestBodyBuilder.buildBodyFromObject(parameters)
+                } else if let count = objectParameters.count where count > 0 {
+                    URLRequest.HTTPBody = try NSJSONSerialization.dataWithJSONObject(objectParameters, options: NSJSONWritingOptions.PrettyPrinted)
```

Comment:
This should be `URLRequest.HTTPBody = try requestBodyBuilder.buildBodyFromObject(objectParameters)`!